packages/affiliates/libs/affiliates/utils.js

```diff
@@ -0,0 +1,281 @@
+// collection of functions to compose a automation pipeline
```

Comment:
this utility file is just a bunch of helper functions that can be mostly unit tested